trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.controller.js

```diff
@@ -1,15 +1,26 @@
 const { format, fromUnixTime, getUnixTime } = require('date-fns');
 const { AMENDMENT_STATUS } = require('@ukef/dtfs2-common');
-const { isTfmFacilityEndDateFeatureFlagEnabled } = require('@ukef/dtfs2-common');
+const { HttpStatusCode } = require('axios');
 const api = require('../../../api');
 const { coverEndDateValidation } = require('./validation/amendCoverEndDateDate.validate');
+const { isFacilityEndDateEnabledForTfmFacility } = require('../../helpers/isFacilityEndDateEnabledForTfmFacility');
+
+const getNextPage = (apiResponseStatus, changeFacilityValue, showFacilityEndDatePage, baseUrl) => {
+  if (apiResponseStatus !== HttpStatusCode.Ok) {
+    throw Error('Error updating amendment');
+  }
+  if (showFacilityEndDatePage) {
```

Comment:
can an empty line be entered between each of these ifs

Reply:
Sure, I've refactored